.env

```diff
@@ -1,2 +1,2 @@
 VITE_API_URL=http://localhost:3001
-VITE_GOOGLE_MAPS_API_KEY=<VITE_GOOGLE_MAPS_API_KEY>
\ No newline at end of file
+VITE_GOOGLE_MAPS_API_KEY=AIzaSyBV4ouFM1BdGprPXoLsW_r9BvBH5ISNPgc
```

Comment:
bitte den key raus nehmen und die datei brauchst du nicht zu commiten. 
Die datei und die package-lock.json file in gitignore schreiben damit es nicht committed wird